.travis.yml

```diff
@@ -1,38 +1,70 @@
-branches:
-  only:
-    - master
 language: objective-c
 os: osx
 osx_image: xcode8.3
-env:
-  global:
-    - LC_CTYPE=en_US.UTF-8
-    - LANG=en_US.UTF-8
-  matrix:
-    - TEST_TYPE=iOS
-    - TEST_TYPE=macOS
-    - TEST_TYPE=Deployment
-    - TEST_TYPE=Carthage
-    - TEST_TYPE=CocoaPods
+branches:
+  only:
+    - master
+    - /^v?[0-9]+.[0-9]+.[0-9]+(-.*)?$/
```

Comment:
You're going to want to escape those `.`s. In a regex it's interpreted as a special 'match any' character unless it's escaped by a preceding `\`. This regex could match stuff like `123` or `1a2b3c` in addition to `v1.2.3`. The exception being if it's in a character class, such as `[.]`, where it's interpreted just as it is.

Reply:
uhm that's a good point! I need to update it all over the place then :)